sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn

```diff
@@ -735,7 +735,7 @@
0;    totalChildCareCost: if is defined(calculatedDataTotalChildCareCost) &#10;    then calculatedDataTotalChildCareCost else null,&#10;    totalNonEducationalCost: if is defined(calaulatedDataTotalNonEducationalCost) &#10;    then calaulatedDataTotalNonEducationalCost else null,&#10;    dependantChildQuantity: if is defined(calculatedDataDependantChildQuantity) &#10;    then calculatedDataDependantChildQuantity else null,&#10;    dependantChildInDaycareQuantity: if is defined(calculatedDataDependantChildInDaycareQuantity)&#10;    then calculatedDataDependantChildInDaycareQuantity else null,&#10;    dependantInfantQuantity: if is defined(calculatedDataDependantInfantQuantity)&#10;    then calculatedDataDependantInfantQuantity else null,&#10;    dependantDeclaredOnTaxesQuantity: if is defined(calculatedDataDependantDeclaredOnTaxesQuantity)&#10;    then calculatedDataDependantDeclaredOnTaxesQuantity else null,&#10;    dependantPostSecondaryQuantity: if is defined(calculatedDataDependantPostSecondaryQuantity)&#10;    then calculatedDataDependantPostSecondaryQuantity else null,&#10;    partnerStudentStudyWeeks: if is defined(calculatedDataPartnerStudentStudyWeeks)&#10;    then calculatedDataPartnerStudentStudyWeeks else null,&#10;    dependants11YearsOrUnder: if is defined(calculatedDataDependants11YearsOrUnder)&#10;    then calculatedDataDependants11YearsOrUnder else null,&#10;    dependants12YearsOverOnTaxes: if is defined(calculatedDataDependants12YearsOverOnTaxes)&#10;    then calculatedDataDependants12YearsOverOnTaxes else null,&#10;    partner1TotalIncome: if is defined(calculatedDataPartner1TotalIncome)&#10;    then calculatedDataPartner1TotalIncome else null,&#10;    studentTotalIncome: if is defined(calculatedDataStudentTotalIncome)&#10;    then calculatedDataStudentTotalIncome else null, &#10;    totalDaycareCosts11YearsOrUnder: if is defined(calculatedDataTotalDaycareCosts11YearsOrUnder)&#10;    then calculatedDataTotalDaycareCosts11YearsOrUnder else null,&#10;    totalDaycareCosts12YearsOrOver: if is defined(calculatedDataTotalDaycareCosts12YearsOrOver)&#10;    then calculatedDataTotalDaycareCosts12YearsOrOver else null,&#10;    totalFederalFSC: if is defined(calculatedDataTotalFederalFSC) then calculatedDataTotalFederalFSC else null,&#10;    totalProvincialFSC: if is defined(calculatedDataTotalProvincialFSC) then calculatedDataTotalProvincialFSC else null,&#10;  exemptScholarshipsBursaries: if is defined(calculatedDataExemptScholarshipsBursaries) then calculatedDataExemptScholarshipsBursaries else null,&#10;  studentSpouseContributionWeeks: if is defined(calculatedDataSpouseContributionWeeks) then calculatedDataSpouseContributionWeeks else null,&#10;      fullTimeBooksCost: if is defined(calculatedDataTotalBookCost) then calculatedDataTotalBookCost else null,&#10;    returnTransportationCost: if is defined(calculatedDataReturnTransportationCost) then calculatedDataReturnTransportationCost else null&#10;  }&#10;}" target="workflowData" />
```

Comment:
For my understanding, what was the motivation of having the variable `fullTimeBooksCost` prefixed with "fullTime"?

Reply:
As we talked, I'm gonna remove the prefix.